src/eko/anomalous_dimensions/__init__.py

```diff
@@ -52,9 +53,9 @@ def exp_singlet(gamma_S):
 
     See Also
     --------
-        eko.anomalous_dimensions.lo.gamma_singlet_0 : :math:`\gamma_{S}^{(0)}(N)`
-        eko.anomalous_dimensions.nlo.gamma_singlet_1 : :math:`\gamma_{S}^{(1)}(N)`
-        eko.anomalous_dimensions.nnlo.gamma_singlet_2 : :math:`\gamma_{S}^{(2)}(N)`
+        eko.anomalous_dimensions.as1.gamma_singlet_0 : :math:`\gamma_{S}^{(0)}(N)`
```

Comment:
So, the final decision has been to don't write the power explicit when it is 0, is it correct?
If yes, for me it's also fine.

Reply:
We were wondering whether to drop also the 1 and 2 from as2 and as3, since once they are scoped there's no need to specify them. What should we do?

Reply:
According to me, drop them all. @felixhekhorn?

Reply:
I would drop them